docs/README.md

```diff
@@ -359,6 +359,25 @@ This command should return the following output if everything is OK:
 ```
 {"metric":{"__name__":"system.load.1","environment":"test","host":"test.example.com"},"values":[0.5],"timestamps":[1632833641000]}
 ```
+Some users want to use list of host tags which can be declared by environment variable `DD_TAGS` or in
+`datadog.yaml` config file by setting tags section like in example below:
+```yaml
+tags:
+  - "team:team"
+  - "dev:dev"
+```
+In that case datadog agent starts to send this tags via `/datadog/intake` endpoint.  
+At that moment it is not present a clear solution for gathering tags from this endpoint.
+As a workaround we suggest to use next schema of components realization:
+
+![DataDog tags workaround scheme](assets/images/datadog.png)
```

Comment:
arrows direction on the image seems inconsistent. It is unclear where scraping and where pushing happens. I suspect, all components there "push" metrics, so all arrows should be directed accordingly.